src/libs/actions/connections/index.ts

```diff
@@ -46,7 +46,7 @@ function updatePolicyConnectionConfig<TConnectionName extends ConnectionName, TS
     policyID: string,
     connectionName: TConnectionName,
     settingName: TSettingName,
-    settingValue?: Partial<Connections[TConnectionName]['config'][TSettingName]>,
+    settingValue: Partial<Connections[TConnectionName]['config'][TSettingName]>,
```

Comment:
Fixed this here because `settingValue` is NOT optional. This was causing backend server errors.